src/toM3u8.js

```diff
@@ -15,6 +15,14 @@ export const formatAudioPlaylist = ({ attributes, segments }) => {
 };
 
 export const formatVttPlaylist = ({ attributes, segments }) => {
+  if (segments === undefined) {
```

Comment:
should probably do `typeof segments === 'undefined'`. Thought that the linter should catch this, but maybe its not setup as a rule